opi/plugins/packman.py

```diff
@@ -31,3 +31,12 @@ def run(cls, query):
 			'gstreamer-plugins-bad',
 			'gstreamer-plugins-ugly',
 		])
+
+		if not opi.ask_yes_or_no("Do you want to install openh264 codecs from openh264 repository?", 'y'):
```

Comment:
```suggestion
		if not opi.ask_yes_or_no("Do you want to install openh264 codecs from openSUSE openh264 repository?", 'y'):
```

Let's point out that the repo is more or less official